TestsCalculator/CalculatorPageTests.cs

```diff
@@ -195,5 +195,30 @@ public void CheckDefaultStartDate()
             //Assert
             Assert.AreEqual(DateTime.Today.ToString("d/M/yyyy"), calculatorPage.StartDate);
         }
+
+        [Test]
+        public void CheckSettingsBtnDisplayed()
+        {
+            //Arrange
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+
+            //Assert
+            Assert.IsTrue(calculatorPage.SettingsBtn.Displayed);
+        }
+
+        [Test]
+        public void CheckSettingsBtnNavigation()
+        {
+            //Arrange
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+
+            //Act
+            calculatorPage.SettingsBtn.Click();
+
+            //Assert
+            string actualUrl = driver.Url;
+            string expectedUrl = "http://127.0.0.1:8080/Settings";
+            Assert.AreEqual(expectedUrl, actualUrl);
```

Comment:
You may add a property to a Settings page-object:
```
bool IsPageOpened => driver.Url.Contains("Settings");
```

```suggestion
            settingsPage = new SettingsPage(driver); // add this to Arrange section
            Assert.IsTrue(settingsPage.IsOpened);
```